builder/dockerfile/metrics.go

```diff
@@ -0,0 +1,29 @@
+package dockerfile
+
+import "github.com/docker/go-metrics"
+
+var (
+	buildsTriggered metrics.Counter
+	buildsFailed    metrics.LabeledCounter
+)
+
+func init() {
```

Comment:
Ideally not because it'll just become a trash-heap of places to put stuff.
And ideally these constants would be namespaced with metrics.
So it'd be something like`metrics.DockerfileSyntaxError`

We can initialize all this stuff there.